app-blank-after-kendo-upgrade.md

```diff
@@ -0,0 +1,43 @@
+
+---
+title: Upgrading from Kendo UI Core to Professional Breaks App
+description: After upgrading an app through the Package Manager to use Kendo UI Professional it appears blank on start up. 
```

Comment:
Comma after Professional.

Why use a comma here - it's a good option to visually (and logically) separate the action and the consequence. 

Also, consider using variables for the Package names. Either page or site-level vars. Site-level vars will work in the frontmatter. It's possible that page-level vars will also work but you need to check this.